tensorflow-xla-runtime.spec

```diff
@@ -0,0 +1,32 @@
+### RPM external tensorflow-xla-runtime 2.12.0
```

Comment:
@riga, we need `tensorflow-xla-runtime` to be part of cmssw dependency chain, so please add it in https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_14_1_X/master/cmssw-tool-conf.spec

Reply:
Added :+1: